atc/db/resource.go

```diff
@@ -337,6 +342,16 @@ func (r *resource) CreateBuild(ctx context.Context, manuallyTriggered bool) (Bui
 		return nil, false, err
 	}
 
+	err = r.conn.Bus().Notify(atc.ComponentBuildTracker)
```

Comment:
Is there a reason we don't notify the build events channel when we start resource/resource type checks, but do when we start "regular" builds? I guess the only time you'd actually want to monitor the build events is when you manually trigger a check, and the first event that comes occurs (presumably the initialize event when the build is picked up) will notify the channel - just wondering why they're different

Reply:
Yeah - the other one notifies because the build already existed, so a user may have the page open. In this case it's a newly created build so there's no one around to notify.